packages/build/src/plugins_core/functions/index.js

```diff
@@ -56,8 +56,11 @@ const getZisiParameters = ({ buildDir, featureFlags, functionsConfig, functionsD
     expression,
     normalizeFunctionConfig({ buildDir, featureFlags, functionConfig: object, isRunningLocally }),
   ])
+  const zisiFeatureFlags = {
+    defaultEsModulesToEsbuild: featureFlags.buildbot_es_modules_esbuild,
```

Comment:
https://github.com/netlify/zip-it-and-ship-it/blob/9b0c6d56e4e05704122fdd55cacc81d75cf40aa1/src/feature_flags.js#L7